File: api/DTOs/HR/ProspectiveBriefDto.cs
namespace api.DTOs.HR
{
    public class ProspectiveBriefDto
    {
        public int Id { get; set; }
        public bool Checked { get; set; }
        public string Source { get; set; }
        public string PersonType { get; set; }
        public string PersonId { get; set; }
        public int ProspectiveCandidateId { get; set; }
        public string Nationality { get; set; }
        public DateTime DateRegistered { get; set; }
        public string CandidateName { get; set; }
        public string CategoryRef { get; set; }
        public string PhoneNo { get; set; }
        public string Email { get; set; }
        public string CurrentLocation { get; set; }
        public string WorkExperience { get; set; }
        public string Status { get; set; }
        public bool BySMS {get; set; }
        public bool ByMail {get; set; }
        public bool ByPhone {get; set; }
        public string Username { get; set; }
        public DateTime StatusDate { get; set; }
        public string Remarks { get; set; }
    }
}

File: client/src/app/_dtos/hr/prospectiveBriefDto.ts
export interface IProspectiveBriefDto
{
    id: number;
    checked: boolean;
    source: string;
    personType: string;
    personId: string;
    nationality: string;
    dateRegistered: Date;
    candidateName: string;
    categoryRef: string;
    customerName: string;
    phoneNo: string;
    email: string;
    currentLocation: string;
    workExperience: number;
    status?: string;
    bySMS: boolean;
    byMail: boolean;
    byPhone: boolean;
    userName: string;
    statusDate: Date;
    remarks: string;
}